diff -urN /home/users/builder/1orig/frotz-2.43/Makefile frotz-2.43/Makefile
--- /home/users/builder/1orig/frotz-2.43/Makefile	Mon Oct 28 02:21:52 2002
+++ frotz-2.43/Makefile	Fri Aug 15 00:25:07 2003
@@ -7,25 +7,25 @@
 # Define your optimization flags.  Most compilers understand -O and -O2,
 # Standard (note: Solaris on UltraSparc using gcc 2.8.x might not like this.)
 #
-OPTS = -O2 
+OPTS = -O2 -I/usr/include/ncurses
 
 # Pentium with gcc 2.7.0 or better
 #OPTS = -O2 -fomit-frame-pointer -malign-functions=2 -malign-loops=2 \
 #-malign-jumps=2
 
 # Define where you want Frotz installed.  Usually this is /usr/local
-PREFIX = /usr/local
+PREFIX = /usr
 
-MAN_PREFIX = $(PREFIX)
+MAN_PREFIX = $(PREFIX)/share
 #MAN_PREFIX = /usr/local/share
 
-CONFIG_DIR = $(PREFIX)/etc
+#CONFIG_DIR = $(PREFIX)/etc
 #CONFIG_DIR = /etc
 
 # Define where you want Frotz to look for frotz.conf.
 #
-CONFIG_DIR = /usr/local/etc
-#CONFIG_DIR = /etc
+#CONFIG_DIR = /usr/local/etc
+CONFIG_DIR = /etc
 #CONFIG_DIR = /usr/pkg/etc
 #CONFIG_DIR =
 
@@ -214,10 +214,10 @@
 
 install: $(NAME)
 	strip $(BINNAME)$(EXTENSION)
-	install -d $(PREFIX)/bin
-	install -d $(MAN_PREFIX)/man/man6
-	install -c -m 755 $(BINNAME)$(EXTENSION) $(PREFIX)/bin
-	install -c -m 644 doc/$(NAME).6 $(MAN_PREFIX)/man/man6
+	install -d $(DESTDIR)$(PREFIX)/bin
+	install -d $(DESTDIR)$(MAN_PREFIX)/man/man6
+	install -c -m 755 $(BINNAME)$(EXTENSION) $(DESTDIR)$(PREFIX)/bin
+	install -c -m 644 doc/$(NAME).6 $(DESTDIR)$(MAN_PREFIX)/man/man6
 
 uninstall:
 	rm -f $(PREFIX)/bin/$(NAME)
